app.py

```diff
@@ -15,22 +14,27 @@
            methods=['POST'],
            content_types=['application/octet-stream'])
 def exportBag():
+    """Expects a compressed (zipped) BagIt structure that
+    contains two TSV files named  "participant.tsv" and
+    "sample.tsv" in its "data" sub-directory. It creates
+    a new workspace in a FireCloud namespace, and uploads
+    the data into it.
+    """
     req_body = app.current_request.raw_body
     req_query_params = app.current_request.query_params
     req_headers = app.current_request.headers
     os.chdir('/tmp')
     with zipfile.ZipFile(io.BytesIO(req_body), 'r') as archive:
         archive.extractall()
-    tsv_fname = '/tmp/manifest_bag/data/manifest.tsv'
+    payload = []
+    payload.append('/tmp/manifest/data/participant.tsv')
```

Comment:
I don't think you can assume the directory name is `manifest`. While there can be a convention for bagits generated by Boardwalk, someone else might generate a bagit with a different directory name. As far as I can tell, the directory name is not part of the spec, and you should be able to handle bagits that are not coming from Boardwalk.